tequilapi/endpoints/transactor.go

```diff
@@ -513,22 +544,32 @@ func (te *transactorEndpoint) TokenRewardAmount(resp http.ResponseWriter, reques
 
 // AddRoutesForTransactor attaches Transactor endpoints to router
 func AddRoutesForTransactor(
-	router *httprouter.Router,
 	identityRegistry identityRegistry,
 	transactor Transactor,
 	promiseSettler promiseSettler,
 	settlementHistoryProvider settlementHistoryProvider,
 	addressProvider addressProvider,
-) {
+) func(*gin.Engine) error {
 	te := NewTransactorEndpoint(transactor, identityRegistry, promiseSettler, settlementHistoryProvider, addressProvider)
-	router.POST("/identities/:id/register", te.RegisterIdentity)
-	router.GET("/transactor/fees", te.TransactorFees)
-	router.POST("/transactor/settle/sync", te.SettleSync)
-	router.POST("/transactor/settle/async", te.SettleAsync)
-	router.GET("/transactor/settle/history", te.SettlementHistory)
-	router.POST("/transactor/stake/increase/sync", te.SettleIntoStakeSync)
-	router.POST("/transactor/stake/increase/async", te.SettleIntoStakeAsync)
-	router.POST("/transactor/stake/decrease", te.DecreaseStake)
-	router.POST("/transactor/settle/withdraw", te.Withdraw)
-	router.GET("/transactor/token/:token/reward", te.TokenRewardAmount)
+
+	return func(e *gin.Engine) error {
+		idGroup := e.Group("/identities")
+		{
+			idGroup.POST("/:id/register", te.RegisterIdentity)
+		}
+
+		transGroup := e.Group("/transactor")
+		{
+			transGroup.GET("/fees", te.TransactorFees)
+			transGroup.POST("/settle/sync", te.SettleSync)
+			transGroup.POST("/settle/async", te.SettleAsync)
+			transGroup.GET("/settle/history", te.SettlementHistory)
+			transGroup.POST("/stake/increase/sync", te.SettleIntoStakeSync)
+			transGroup.POST("/stake/increase/async", te.SettleIntoStakeAsync)
+			transGroup.POST("/stake/decrease", te.DecreaseStake)
+			transGroup.POST("/settle/withdraw", te.Withdraw)
+			transGroup.GET("/transactor/token/:token/reward", te.TokenRewardAmount)
```

Comment:
This will probably also produce a path something like `/transactor/transactor/token/:token/reward`